tests/functional/spec/features/validation/sum/equal_total_in_separate_section_hub.spec.js

```diff
@@ -0,0 +1,179 @@
+      answerAndSubmitEmployeeBreakdownQuestion(10, 5);
+
+      expect($(EmployeesBreakdownPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 10");
+    });
+  });
+
+  describe("Given I start a grouped answer validation with dependent sections and complete the overview and breakdown sections", () => {
+    before(() => {
+      browser.openQuestionnaire("test_validation_sum_against_total_hub_with_dependent_section.json");
+
+      // Complete overview section
+      answerAndSubmitTotalTurnoverQuestion(1000);
+      answerAndSubmitTotalEmployeesQuestion(10);
+      $(CompanySectionSummary.submit()).click();
+
+      // Complete breakdown section
+      $(HubPage.submit()).click();
+      answerAndSubmitTurnoverBreakdownQuestion(500, 250, 250);
+      answerAndSubmitEmployeeBreakdownQuestion(5, 5);
+      $(BreakdownSectionSummary.submit()).click();
+
+      expect($(HubPage.summaryRowState(breakdownSectionId)).getText()).to.equal("Completed");
+    });
+
+    it("When I change my answer for the total turnover question, Then the breakdown section should be marked as 'Partially completed'", () => {
+      $(HubPage.summaryRowLink(companyOverviewSectionID)).click();
+      $(CompanySectionSummary.totalTurnoverAnswerEdit()).click();
+
+      answerAndSubmitTotalTurnoverQuestion(1500);
+      $(CompanySectionSummary.submit()).click();
+      expect($(HubPage.summaryRowState(breakdownSectionId)).getText()).to.equal("Partially completed");
+    });
+
+    it("When I click 'Continue with section' on the breakdown section, Then I should be taken to the turnover breakdown question and my previous answers should be prefilled", () => {
+      $(HubPage.summaryRowLink(breakdownSectionId)).click();
+
+      expect($(TurnoverBreakdownPage.turnoverBreakdown1()).getValue()).to.equal("500.00");
+      expect($(TurnoverBreakdownPage.turnoverBreakdown2()).getValue()).to.equal("250.00");
+      expect($(TurnoverBreakdownPage.turnoverBreakdown3()).getValue()).to.equal("250.00");
+    });
+
+    it("When I submit the turnover breakdown question with no changes, Then I should see a validation error", () => {
+      $(TurnoverBreakdownPage.submit()).click();
+
+      expect($(TurnoverBreakdownPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to £1,500.00");
+    });
+
+    it("When I update my answers to equal the new total turnover, Then I should be able to get to the section summary and the breakdown section should be marked as 'Completed'", () => {
+      answerAndSubmitTurnoverBreakdownQuestion(500, 500, 500);
+
+      expect(browser.getUrl()).to.contain(BreakdownSectionSummary.pageName);
+      $(BreakdownSectionSummary.submit()).click();
+      expect($(HubPage.summaryRowState(breakdownSectionId)).getText()).to.equal("Completed");
+    });
+
+    it("When I change my answer for the total employees question, Then the breakdown section should be marked as 'Partially completed'", () => {
```

Comment:
I don't think we need a functional tests for both turnover and employees as it is doing the same thing, doing it at unit level for both is more appropriate. This would also follow are pragmatic approach of having functional tests when needed